src/widgets/stats/stats.tsx

```diff
@@ -139,7 +139,7 @@ const stats: StatsWidget = widgetParams => {
   const {
     container,
     cssClasses: userCssClasses = {} as StatsCSSClasses,
-    templates = defaultTemplates,
+    templates: userTemplates = {} as StatsTemplates,
```

Comment:
In some other widget we were able to remove the `as StatsTemplates` cast. Why not here?

Reply:
Good catch, this is indeed not needed: https://github.com/algolia/instantsearch.js/pull/4785/commits/e95210c1a0568ad92493815760bc6dc9a8464f0b